setup/monitoring/index.md

```diff
@@ -6,3 +6,262 @@ sidebar:
+However when considering all the distinct time-series streams from the
+different label values there are several thousand distinct streams. Some
+metrics are tagged with the application or account they were used on
+behalf of, so the number of streams may grow as the scope of your
+deployment grows. Typically you will be aggregating these dimensions
+together while breaking out along others. The granularity can come in
+handy when it comes time to diagnose problems or investigate for deeper
+understanding of runtime behaviors but you can aggregate across dimensions
+(or parts of dimensions) when you dont care about that level of refinement.
+
+Each microservice defines and exports its own metrics. The
+`spinnaker-monitoring` daemon augments these by identifying which
+microservice they came from (how it does so depends on the backend
+metrics system you are using). There are no global "Spinnaker" metrics.
+There are only individual metrics on the individual instance of
+concrete microservices that, taken together, compose a "Spinnaker" deployment.
+
+
+### Types of Metrics
+
+There are two basic types of metrics currently supported,
+*counters* and *gauges*.
+
+  * __Counters__ are monotonically increasing values over the lifetime of
+    the process. The process starts out with them at 0, then increments
+    them as appropriate. Some counters may increase by 1 each time, such
+    as the number of calls. Other counters may increase by an arbitrary
+    (but non-negative) amount, such as number of bytes.
+
+    Counters are scoped to the process they are in. If you have a counter
+    in each of two different microservice replicas (including a restart),
+    those counters will be independent of one another. Each process only
+    knows about itself. The daemon adds a tag to each data point that
+    identifies which instance it came from so that you can drill down
+    into individual instances if you need. However, typically you will
+    use your monitoring system to aggregate counters across all replicas.
+
+    Counters are useful to determine rates. Given two points in time,
+    the counter differences will be the measurement delta and the
+    delta divided by the time difference will be the rate.
+    (divide by another 1000000 to convert nanoseconds to milliseconds).
+
+    * Spinnaker has a special type of counter called a *Timer*.
+
+      __Timers__ are used to measure timing information. These are
+      always in nanoseconds. When consuming metrics straight from
+      Spinnaker, a Timer will have two complementary time series.
+      One will have a tag "statistic" with the value "count" and
+      the other a tag with a "statistic" with the value "totalTime".
+
+      The Monitoring Daemon transforms these into two distinct
+      counters and removes the "statistic" label entirely. The
+      "count" counter is named with the original metric name plus
+      a "__count" suffix. The "totalTime" with a "__totalTime" suffix.
```

Comment:
If the fact that "statistic" is not relevant to the end user, I wouldn't even mention it. You can add this bit to the Reference section and mention that the daemon doesn't use it for anything, so anyone writing their own direct scraping impl knows they can drop it too.